services/autoscaling/src/simcore_service_autoscaling/modules/auto_scaling_core.py

```diff
@@ -73,6 +78,7 @@ async def _analyze_current_cluster(
     existing_ec2_instances = await get_ec2_client(app).get_instances(
         key_names=[app_settings.AUTOSCALING_EC2_INSTANCES.EC2_INSTANCES_KEY_NAME],
         tags=auto_scaling_mode.get_ec2_tags(app),
+        state_names=["pending", "running"],
```

Comment:
dont you have some enums or literals in one place for these state names?

Reply:
ok, so get_instances takes state_names which is a InstanceTypeType or so which defines the states, which are literals. so all good.